packages/neptune-extension/config/webpack.common.js

```diff
@@ -73,6 +73,8 @@ module.exports = function ({ mode, platform, project }) {
        */
       new webpack.DefinePlugin({
         'NEPTUNE_BUILD_DATE': JSON.stringify(new Date()),
+        'PLATFORM': JSON.stringify(platform),
+        'APP_ENV': JSON.stringify(mode),
```

Comment:
This is usually called `NODE_ENV` by convention.

Reply:
NODE_ENV broke build. It was previously set by Maciej.